lib/screens/create_region_screens/interactor/mappers/create_region_state_mapper.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:seeds/datasource/remote/model/transaction_response.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/create_region_screens/interactor/viewmodels/create_region_bloc.dart';
+import 'package:seeds/screens/create_region_screens/interactor/viewmodels/create_region_page_commands.dart';
+
+class CreateRegionStateMapper extends StateMapper {
+  CreateRegionState mapResultToState(CreateRegionState currentState, Result<TransactionResponse> result) {
+    if (result.isError) {
+      return currentState.copyWith(
+          pageState: PageState.success, pageCommand: ShowErrorMessage("Error Creating Region"));
+    } else {
+      print(' Transaction ID: ${result.asValue!.value.transactionId}');
```

Comment:
Do we need print the transaction id here ?

Reply:
I think the transaction Id should be somewhere in the UI, will ask Nik and Json, but for now I think this is the best place to have it print. Please let me know if you think it should be printed somewhere else